chapters/en/Unit 2 - Convolutional Neural Networks/yolo.mdx

```diff
@@ -0,0 +1,73 @@
+
+### **YOLOv3 (2018)**
+
+YOLOv3[2] again improved significantly the detection speed and accuracy by replacing the Darknet-19 architecture with the more complex but efficient Darknet-53. Also, it dealt more with the localization problem by using three different scales for object detection (13x13, 26x26, and 52x52 grids). This helped find objects with different sizes in the same area. This increased the bounding boxes to: 13 x 13 x 3 + 26 x 26 x 3 + 52 x 52 x 3 = 10,647. The Non-Maximum Suppression (NMS) was still used to filter out redundant overlapping boxes.
+
+### **YOLOv4 (2020)**
+
+Back in 2020, YOLOv4[3] became one of the best detection models in terms of speed and accuracy. The authors changed again the backbone architecture to the faster and better accuracy CSPDarknet53[4]. An important improvement of this version was the optimization for efficient resource utilization, making it suitable for deployment on various hardware platforms, including edge devices. Also, It included a number of augmentations before training that further improved the model prediction accuracy on custom data.
+
+### **YOLOv5 (2020)**
+
+YOLOv5[5] translated the Darknet framework (written in C) to the more flexible and ease of use Pytorch framework. This version automated the previous anchor detection mechanism introducing the auto-anchors. Auto-anchors train the model anchors automatically to match your data. During training, YOLO automatically uses k-means initially and genetic methods to evolve the new better-matched anchors and place them back into the YOLO model. Also, it offered different types of models that depend on the hardware constraints with names similar to today’s YOLOv8 models: YOLOv5s, YOLOv5m, YOLOv5l, and YOLOv5x.
+
+### **YOLOv6 (2022)**
+
+The next version, YOLOv6[6][7], was released by the Meituan Vision AI Department under the article title: "YOLOv6: A Single-Stage Object Detection Framework for Industry." This team made further improvements in terms of speed and accuracy by focusing on five aspects:
+1) Reparameterization using the RepVGG technique. Which is a modified version of VGG with skip connections. During inference, these connections are fused to improve the speed.
+2) Quantization of reparameterization-based detectors. Which added blocks that are called Rep-PANs.
+3) Recognition of the importance of considering different hardware costs and capabilities for model deployment.
```

Comment:
```suggestion
3) Recognition of the importance of considering different hardware costs and capabilities for model deployment.
```
I do not understand what you meant

Reply:
As the [paper ](https://arxiv.org/pdf/2209.02976) mentions: 

_Previous works tend to pay less attention
to deployment, whose latencies are commonly compared
on high-cost machines like V100. There is a hardware gap
when it comes to real serving environment. Typically, lowpower GPUs like Tesla T4 are less costly and provide rather
good inference performance._